### PR TITLE
Allow adding SSH keys even if SSH server is disabled

### PR DESCRIPTION
**Pull request by dsseng**
This is useful in case you want to use them for signature verification.

A notice is added to avoid confusion.

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -739,6 +739,7 @@ principal_state_desc = This principal has been used in the last 7 days
 show_openid = Show on profile
 hide_openid = Hide from profile
 ssh_disabled = SSH Disabled
+ssh_signonly = Notice: SSH is disabled, these keys will only be used for verifying commit signatures
```
Comment (silverwind) [inline options/locale/locale_en-US.ini]:
```suggestion
ssh_signonly = SSH is disabled, these keys will only be used for verifying commit signatures
```

Don't need a prefix

Comment (dsseng) [reply options/locale/locale_en-US.ini]: Thanks

File: templates/user/settings/keys_ssh.tmpl [hunk 2]
```diff
@@ -35,6 +31,11 @@
 		<div class="item">
 			{{.locale.Tr "settings.ssh_desc"}}
 		</div>
+		{{if .DisableSSH}}
+			<div class="item">
```
Comment (silverwind) [inline templates/user/settings/keys_ssh.tmpl:35]: Likely should be a warning message, e.g. https://fomantic-ui.com/collections/message.html#warning

Please show a screenshot.

Comment (dsseng) [reply templates/user/settings/keys_ssh.tmpl:35]: Unlikely to require a warning level, however I'm not sure. 

I can send a screenshot tomorrow

Comment (silverwind) [reply templates/user/settings/keys_ssh.tmpl:35]: Yeah, I haven't tested yet. Depends on UI context if a warning box is warranted. Likely it's not something the user can do anything against, so it's at most a "info" box I guess.

Comment (dsseng) [reply templates/user/settings/keys_ssh.tmpl:35]: grafik

File: options/locale/locale_en-US.ini [hunk 3]
```diff
@@ -739,6 +739,7 @@ principal_state_desc = This principal has been used in the last 7 days
 show_openid = Show on profile
 hide_openid = Hide from profile
 ssh_disabled = SSH Disabled
+ssh_signonly = SSH is disabled, these keys will only be used for verifying commit signatures
```
Comment (silverwind) [inline options/locale/locale_en-US.ini]:
```suggestion
ssh_signonly = SSH is currently disabled so these keys are only used for commit signature verification.
```

File: templates/user/settings/keys_ssh.tmpl (hunk 2 above)
Comment (silverwind) [reply templates/user/settings/keys_ssh.tmpl:35]: Okay, seems good without any "severity" there.

**Review by silverwind — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**